fenix/app/src/main/java/org/mozilla/fenix/compose/Menu.kt

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.compose
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.DropdownMenu
+import androidx.compose.material.DropdownMenuItem
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.DisposableEffect
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Popup action menu.
+ *
+ * @param showMenu Whether or not the menu is currently displayed to the user.
+ * @param onDismissRequest To be called when user requests to dismiss the menu (on rotation change,
+ * on outside click, on back pressed, on interacting with any item).
+ * @param menuItems List of items to be displayed in the menu.
+ * @param menuItems List of items to be displayed in the menu.
+ */
+@Composable
+fun Menu(
```

Comment:
Perhaps we should call this `DropdownMenu` unless this already follows the design component naming? wdyt @MozillaNoah?

Reply:
I agree. It was probably named `Menu` before I updated the ticket name. This will also make it easier to differentiate the vertical dropdown from the row-based menu in the figma, whenever that is developed.